lightning/src/chain/channelmonitor.rs

```diff
@@ -1357,10 +1357,13 @@ impl<Signer: Sign> ChannelMonitorImpl<Signer> {
 		// *we* sign a holder commitment transaction, not when e.g. a watchtower broadcasts one of our
 		// holder commitment transactions.
 		if self.broadcasted_holder_revokable_script.is_some() {
-			let (claim_reqs, _) = self.get_broadcasted_holder_claims(&self.current_holder_commitment_tx, 0);
+			// Assume that the broadcasted commitment transaction confirmed in the current best
+			// block. Even if not, its a reasonable metric for the bump criteria on the HTLC
```

Comment:
I think this comment is a bit confusing, bumping timer is only function of current height and expiration timelock (see `get_height_timer` in `package.rs`, not of parent transaction conf. So leveraging the best known height as comparison point to refresh timers is the best we can do ?

Reply:
`get_broadcasted_holder_claims` sets the `PackageTemplate::soonest_conf_deadline` to the height given here, so it *is* the "expiration timelock".

Reply:
Yes in this case parent conf height is equal to `PackageTemplate::soonest_conf_deadline` which is equal to "expiration timelock" but my point rather than relying on parent conf height, which have the side-effect of artificially increasing the fee-bump frequency and thus bleed feerate a bit, we should rely on the per-holder-claims timelocks themselves to fulfill `soonest_conf_deadline`. 

Though I don't think this comment needs further action for now and sounds to be related just after.

Reply:
I'm still a bit confused here, but I'll leave it for a followup.